plugins/Dashboard/javascripts/widgetMenu.js

```diff
@@ -145,7 +145,9 @@ widgetsHelper.getWidgetNameFromUniqueId = function (uniqueId, callback) {
  */
 widgetsHelper.loadWidgetAjax = function (widgetUniqueId, widgetParameters, onWidgetLoadedCallback, onWidgetErrorCallback) {
     var disableLink = broadcast.getValueFromUrl('disableLink');
-    if (disableLink.length) {
+    if (disableLink.length
+        || $('body#standalone').length
+    ) {
         widgetParameters['disableLink'] = disableLink;
```

Comment:
That doesn't work correctly. If `$('body#standalone').length` is `true`, but `disableLink` isn't set, it will append `disableLink`, which is empty. Guess that need to be something like `widgetParameters['disableLink'] = disableLink || $('body#standalone').length`